pom.xml

```diff
@@ -141,6 +141,8 @@
     <version.org.apache.karaf>2.4.0</version.org.apache.karaf>
     <version.org.apache.lucene>6.6.1</version.org.apache.lucene>
     <version.org.apache.maven>3.3.9</version.org.apache.maven>
+    <version.org.apache.maven.utils>3.2.1</version.org.apache.maven.utils>
+    <version.org.apache.maven.archiver>3.3.0</version.org.apache.maven.archiver>
```

Comment:
why this version? latest one is `3.5.0`

Reply:
@elguardian can you please comment on this?

Reply:
btw we upgraded maven to 3.6.3 for all jobs, so it would be convenient to check if 3.6.x is available for any maven dependencies you use and try to upgrade